### PR TITLE
[sailfish-webview] Adapt to sharing UI and API changes. JB#53987 OMP#JOLLA-75

### PR DESCRIPTION
**Pull request by blammit**
The sharing UI is now launched out-of-process, so the SharePage-based
components are no longer necessary.

### CODE REVIEW

**Comment by blammit** [comment]
Note: associated API changes need to be merged before this can be merged.

File: import/popups/ShareLinkPage.qml [hunk 1]
```diff
@@ -1,32 +0,0 @@
...
-
-SharePage {
-    id: page
-
-    property string link
-    property string linkTitle
-
-    //: List header for link sharing method list
-    //% "Share link"
-    header: qsTrId("sailfish_components_webview_popups-he-share_link")
```
Comment (chriadam) [inline import/popups/ShareLinkPage.qml:24]: unlikely, but just checking: did something else use this translation ID without specifying the EE translation and context?

Comment (blammit) [reply import/popups/ShareLinkPage.qml:24]: Not within this repo, at least.

**Comment by pvuorela** [comment]
Jolla-email MessageViewFooter using ShareTextPage commented elsewhere, but per se should get removed here.

For ShareLinkPage there's also ContextMenu.qml in this repository.

**Comment by blammit** [comment]
Added WebShareAction to replace ShareLinkPage and ShareTextPage.

**Comment by pvuorela** [comment]
LGTM

**Review by pvuorela — APPROVED** [review]

**Review by atatarov — APPROVED** [review]

**Review by okodron — APPROVED** [review]

**Pull request merged**